### PR TITLE
Addresses #2059 Add logical definition and synonym to lung megakaryocyte

### LINKED ISSUE #2059: Revise logical definition and add synonyms to lung megakaryocyte
**CL term**
[lung megakaryocyte](http://purl.obolibrary.org/obo/CL_4033018)

**Suggested revision of logical definition**
megakaryocyte and ('part of' some 'lung connective tissue') (it was a general class axiom until now)

**Synonyms**
Add synonyms from PMID:[36524131](https://pubmed.ncbi.nlm.nih.gov/36524131):
lung-resident megakaryocyte (exact)
lung resident megakaryocyte (exact)
MKL (RELATED, ABBREVIATION)

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2059 Add logical definition and synonym to lung megakaryocyte

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31393,8 +31393,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33351116") obo:IAO_00001
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0001-6677-8489>)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0002-9185-3994>)
 AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033018 "2023-03-16T11:31:23Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36524131") oboInOwl:hasExactSynonym obo:CL_4033018 "lung resident megakaryocyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36524131") oboInOwl:hasExactSynonym obo:CL_4033018 "lung-resident megakaryocyte")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31397]: Based on the definitions, one might expect 'lung(-) resident megakaryocyte' to be the label.

**Review by bvarner-ebi — APPROVED** [review]
See inline comment for your consideration.

Approved whether label is changed or not.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:31397]: Thank you @bvarner-ebi for the comment. I will ask the requester what is the preferred term.

**Comment by aleixpuigb** [comment]
I have a request from the lung community to update the labels for multiple terms. Once the requester of this term answers, I will create the PR and address it there if adequate.

**Pull request merged**